src/Files.App/Helpers/AppSystemBackdrop.cs

```diff
@@ -88,11 +103,36 @@ private void OnSettingChanged(object? sender, Shared.EventArguments.SettingChang
 					};
 
 				case BackdropMaterialType.Acrylic:
-					return new DesktopAcrylicController();
+					var acrylicController = new DesktopAcrylicController();
+					SetAcrylicBackdropProperties(acrylicController, theme);
+					return acrylicController;
 
 				default:
 					return null;
 			}
 		}
+
+		private void SetAcrylicBackdropProperties(DesktopAcrylicController controller, SystemBackdropTheme theme)
```

Comment:
What happens if the color mode is changed at run time?

Reply:
It changes automatically, see https://github.com/files-community/Files/blob/c5a627f6012e8c5e664d6e84ca567ad0e643ce9e/src/Files.App/Helpers/AppSystemBackdrop.cs#L54